pkg/parser/extended_syntax_parser.y

```diff
@@ -110,6 +112,16 @@ type EvaluateClause struct {
 	EvaluateInto  string
 }
 
+type MathProgClause struct {
+	// Sense can be MAXIMIZE or MINIMIZE
+	Sense string
+	Objective *Expr
+	Constrants ExprList
+	SolveAttrs Attributes
```

Comment:
I don't get what are Solve Attributes. Is it SolverAttrs?

Reply:
Changed to `OptimizeAttrs`